perun-cli/Perun/beans/HashedGenData.pm

```diff
@@ -0,0 +1,304 @@
+package Perun::beans::HashedGenData;
+
+use strict;
+use warnings;
+
+use Perun::Common;
+use Data::Dumper;
+
+sub new
+{
+	bless({});
+}
+
+sub fromHash
+{
+	my $class = shift;
+	$class = ref $class if ref $class;
+
+	my $self = { };
+	my $attributes = { };
```

Comment:
Unused variables attributes and hierarchy.

Reply:
Fixed.